go/pkg/casng/upload_digester.go

```diff
@@ -273,9 +274,9 @@ func (u *uploader) digest(req UploadRequest) {
 				return true
 
 			case info.Mode().IsRegular():
-				stats.DigestCount++
-				stats.InputFileCount++
-				node, blb, errDigest := u.digestFile(realPath, info, req.digestOnly, req.id, req.tag, walkID)
+				stats.DigestCount += 1
+				stats.InputFileCount += 1
```

Comment:
why does it revert https://github.com/bazelbuild/remote-apis-sdks/commit/b2dd45919ae77c3c83953654a1b08660a599319c ?

Reply:
Bad rebase..